Contents/mods/Perdition/media/scripts/atm_MoneyTrader_recipes.txt

```diff
@@ -0,0 +1,125 @@
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+    recipe Money from Watches
+        {
+        destroy WristWatch_Right_ClassicBlack/WristWatch_Left_ClassicBlack/WristWatch_Right_ClassicBrown/WristWatch_Left_ClassicBrown/WristWatch_Right_ClassicMilitary/WristWatch_Left_ClassicMilitary/WristWatch_Right_DigitalBlack/WristWatch_Left_DigitalBlack/WristWatch_Right_DigitalRed/WristWatch_Left_DigitalRed/WristWatch_Right_DigitalDress/WristWatch_Left_DigitalDress/,
+        keep MoneyTrader,
+
+        Result:Money=3,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+}
+    recipe Money from Dogtag
+        {
+        destroy Necklace_Dogtag,
+        keep MoneyTrader,
+
+        Result:Money=5,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+    recipe Money from Metal Scraps
+        {
+        destroy ScrapMetal,
+        keep MoneyTrader,
+
+        Result:Money=2,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+    recipe Money from Electronic Scraps
+        {
+        destroy ElectronicsScrap,
+        keep MoneyTrader,
+
+        Result:Money=2,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+    recipe Money from Gunpowder
+        {
+        destroy GunPowder=10,
+        keep MoneyTrader,
+
+        Result:Money=2,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+    recipe Money from Wallet
+        {
+        destroy Wallet/Wallet2/Wallet3/Wallet4,
+        keep MoneyTrader,
+
+        Result:Money=30,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+    recipe Money from Credit Card
+        {
+        destroy CreditCard,
+        keep MoneyTrader,
+
+        Result:Money=200,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+		/*OnCreate:ChangeMineral_2_to_1_OnCreate,*/
+        OnTest:Change_All_For_Money_OnTest,
+		/*RemoveResultItem:true,*/
+        }
+    recipe Money from Medicine
+        {
+        destroy PillsBeta=10/PillsSleepingTablets=10/Pills=10/PillsVitamins=10,
```

Comment:
Why 10 for the delta?  Do you want 10 of the medical item removed or 10 uses?

```suggestion
destroy PillsBeta/PillsSleepingTablets/Pills/PillsVitamins=10,
```

You can't specify more than 1 usedelta when making an alternate item list